src/EditorFeatures/TestUtilities/StubVsServiceExporter`2.cs

```diff
@@ -34,7 +34,7 @@ public StubVsServiceExporter(
         [Import(typeof(SAsyncServiceProvider))] IAsyncServiceProvider2 asyncServiceProvider,
```

Comment:
@davkean's bug specifically calls out IAsyncServiceProvider3, do we need to move to those newer interface here or are we good? His comment makes me think all the methods will eventually call the right thing.

Reply:
My understanding is that the implementations on each of the interfaces will do the right thing, and that there is no need to explicitly use the "3" interface.